.claude/skills/ai-content-pipeline/Skill.md

```
@@ -88,123 +76,268 @@ venv\Scripts\ai-content-pipeline list-models
+from video_utils.ai_commands import cmd_detailed_timeline_with_params
+
+result = cmd_detailed_timeline_with_params(
+    input_path='path/to/video.mp4',
+    output_path='output/',
+    provider='fal',  # or 'gemini'
+    model='google/gemini-3-pro-preview'
+)
+```
+
+**Available Models for Video Analysis:**
+| Provider | Model | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)